### ISSUE TITLE
Report app versions in the output

### ISSUE THREAD

**Issue by eadmaster** [MuhammedKpln/chob#10]
example output:
````
chob vlc --enableExperiementalFeatures

🔎
Searching on AppImage feed..
Searching on Flathub..
Searching on Snapcraft..
Found 3 applications, please choose which one you want.
0) VLC - AppImage
1) VLC v3.1.0 - Flathub
2) VLC v3.0.4 - Snapcraft
````

**Comment by MuhammedKpln** [comment]
Great feature! will implement this but as for appimage there is no version support on their api, so this feature will only work on Flatpak and Snap applications.

**Comment by MuhammedKpln** [comment]
Published new version [v0.6](https://github.com/MuhammedKpln/chob/releases/tag/0.6).
It will now output the application version beside application name.
